multiply_ui/server/context.py

```diff
@@ -2,9 +2,20 @@
 
 from .model import Job
 
+import multiply_data_access.data_access_component
+
 class ServiceContext:
     def __init__(self):
         self._jobs = {}
+        self.data_access_component = multiply_data_access.data_access_component.DataAccessComponent()
+        self._get_mundi_datastore()
+
+    def _get_mundi_datastore(self):
+        for data_store in self.data_access_component._data_stores:
+            if data_store._id == "Mundi":
+                self.data_access_component._data_stores = [data_store]
```

Comment:
Weird.

Reply:
Absolutely. See previous comment.

Reply:
You might also simply remove all other entries from the data_stores.yaml file in the .multiply-dir.